aplt/scenarios.py

```diff
@@ -57,30 +131,66 @@ def basic_forever(notif_delay=300, run_once=0):
         yield ack(channel_id=notif["channelID"], version=notif["version"])
         yield counter("notification.ack", 1)
         yield wait(notif_delay)
+
         if run_once:
+            yield unregister(reg["channelID"])
             yield disconnect()
             break
 
 
-def connect_and_idle_forever():
+def notification_forever_stored(qty_stored=1000, ttl=300, flood_delay=30,
```

Comment:
I'll prob need to tweak these delays if we want stored notifications to be delivered before TTL is over -or- if we want them to be delivered in a flood (i.e. flood_delay=0 for this scenario)